test/test-network/systemd-networkd-tests.py

```diff
@@ -117,8 +118,13 @@ def remove_log_file(self):
         if os.path.exists(dnsmasq_log_file):
             os.remove(dnsmasq_log_file)
 
-    def start_networkd(self):
-        subprocess.check_call('systemctl restart systemd-networkd', shell=True)
+    def start_networkd(self, remove_state_file=False):
+        if (remove_state_file and os.path.exists(os.path.join(networkd_runtime_directory, 'state'))):
+            subprocess.check_call('systemctl stop systemd-networkd', shell=True)
+            os.remove(os.path.join(networkd_runtime_directory, 'state'))
+            subprocess.check_call('systemctl start systemd-networkd', shell=True)
+        else:
```

Comment:
even for LLDP and DHCP should not we clean everything ? I think this this should be by default.

Reply:
I am not sure, but relevant links are always removed. So, I expect it is safe.
If these files disturbs tests, then let's remove them on another PR.